requirements.txt

```
@@ -1,4 +1,4 @@
-aiohttp[speedups]>=3.7.4
+aiohttp>=3.9.2
```

```suggestion
aiohttp[speedups]>=3.9.2
```

Not sure why snyk took that part out.